setup-bootstrap.sh

```diff
@@ -1,22 +1,63 @@
-#! /bin/bash
+#!/usr/bin/env bash
+# Copyright (c) 2022, Linaro. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
 
-set -e
+# Exit on command failure
+set -o errexit
+
+# Fail on unset variable
+# Use "${VARNAME-}" instead of "$VARNAME" to access unset variable(s)
+set -o nounset
+
+# Enable debug mode if $TRACE is set
+# To enable, run with: "TRACE=1 ./script.sh"
```

Comment:
BTW, if you make this `env TRACE=1 ./script.sh` it will work with any shell.

Reply:
Fixed, thanks